pkg/target/dto_util.go

```diff
@@ -61,3 +61,12 @@ func CreateTransactionCommodity(key string, qps *Resource, ctype proto.Commodity
 		Used(qps.Used).
 		Create()
 }
+
+func CreateResponseTimeCommodity(key string, qps *Resource, ctype proto.CommodityDTO_CommodityType) (*proto.CommodityDTO, error) {
```

Comment:
Since the  two functions `CreateResponseTimeCommodity` and `CreateResponseTimeCommodity ` are same, we can delete one, and rename the other.